custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -86,6 +88,12 @@ def _base_properties(self, repeat_record):
             "IP_FROM": server_ip,
         }
 
+    def use_new_2b_app_structure(self, person_case, episode_case):
```

Comment:
do you think it would be useful to just call this `case_version` and do something like:
```py
if (
            episode_case.dynamic_case_properties().get('episode_type') == DSTB_EPISODE_TYPE and
            person_case.dynamic_case_properties().get('case_version') == NEW_2B_APP_PERSON_CASE_VERSION
        ):
    return V2
else:
    return V1
```
It might be a bit more future proof. Willing to be convinced that this is "YAGNI" though.